etstool.py

```diff
@@ -224,7 +224,7 @@ def install(runtime, toolkit, environment, editable, source):
     install_traitsui = "edm run -e {environment} -- pip install "
     if editable:
         install_traitsui += "--editable "
-    install_traitsui += "."
+    install_traitsui += ". --no-deps"
```

Comment:
Nit: It might read better if this `--no-deps` is added after `pip install ` on line 224.

I am tempted to add `--force-reinstall` here just to be safe so we don't recreate what happens in enable where the CI is run against released traitsui code rather than the local source.

Reply:
moved `--no-deps` and added `--force-reinstall`